.devcontainer/devcontainer.json

```diff
@@ -26,5 +31,6 @@
     "cpus": 4,
     "memory": "8gb",
     "storage": "32gb"
-  }
+  },
+  "postAttachCommand": "make local-deps && make vendor && (cd ui/web-v2 && yarn)"
```

Comment:
`make update-repos` might be better than `make vendor`, it will `tidy mod` before generating `vender` directory.

Reply:
sounds good 👍 Thanks!

Reply:
Fixed https://github.com/bucketeer-io/bucketeer/pull/1008/commits/3e3495be96b94820215771445b5dd6ddd20c86e7